### PR TITLE
feat: [PLG-2547]: Added GCP and AWS UI Swimlanes with commands

### PR DESCRIPTION
**Pull request by devinder-harness**
### Summary

Added support for:
* GCP 1st and 2nd Gen Function
* AWS and Serverless.com Lambda
JIRA: https://harness.atlassian.net/browse/PLG-2547
Screenshots
NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by vikrant-harness — DISMISSED** [review]

**Review by bhavya-sinha — APPROVED** [review]

**Review by reetika04 — DISMISSED** [review]

**Review by reetika04 — APPROVED** [review]

**Review by vikrant-harness — APPROVED** [review]

File: src/modules/10-common/utils/JSONUtils.ts [hunk 1]
```diff
@@ -48,4 +48,32 @@ const sanitize = (obj: Record<string, any>, sanityConfig?: YamlSanityConfig): Re
...
+  try {
+    const blob = JSON.stringify(jsonData)
+    const file = new Blob([blob], { type: 'application/json;charset=utf-8' })
+    const data = URL.createObjectURL(file)
+    const anchor = document.createElement('a')
+    anchor.style.display = 'none'
+    anchor.href = data
+    anchor.download = fileName
+    anchor.click()
+    // For Firefox
```
Comment (tan-nhu) [inline src/modules/10-common/utils/JSONUtils.ts:68]: 👍

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Previews/WhereAndHowToDeployPreview.tsx [hunk 2]
```diff
@@ -62,7 +63,7 @@ const DelegateLoadingLabel = ({ data }: { data: WhereAndHowToDeployType }): JSX.
           {getString('deploymentTypeText')}:
         </Text>
         <Text padding={{ left: 'small' }} color={Color.BLACK}>
-          {data.type?.label}
+          {getString(data.type?.label as keyof StringsMap)}
```
Comment (tan-nhu) [inline src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Previews/WhereAndHowToDeployPreview.tsx:66]: This is risky, when `data.type.label` is changed unintentionally, `getString` throws an exception and the app crashes.

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Previews/WhereAndHowToDeployPreview.tsx [hunk 3]
```diff
@@ -97,7 +98,7 @@ const DelegateSuccessLabel = ({
...
```
Comment (tan-nhu) [inline src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Previews/WhereAndHowToDeployPreview.tsx:101]: Same as above

**Review by abhinavrastogi-harness — APPROVED** [review]

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Previews/WhereAndHowToDeployPreview.tsx (hunk 2 above)
Comment (devinder-harness) [reply src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Previews/WhereAndHowToDeployPreview.tsx:66]: sure @tan-nhu will take care in upcoming PRs

**Pull request merged**